fe/fe-core/src/main/java/com/starrocks/connector/paimon/PaimonPartitionInfo.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package com.starrocks.connector.paimon;
+
+import com.starrocks.connector.PartitionInfo;
+
+public class PaimonPartitionInfo implements PartitionInfo {
+
+    private final String partitionName;
+    private final long version;
```

Comment:
I found that most implementation of `PartitionInfo` are almost same but invented again and again